data-prepper-expression/src/main/antlr/DataPrepperExpression.g4

```diff
@@ -121,8 +121,13 @@ regexPattern
     | String
     ;
 
-setInitializer
-    : LBRACE primary (SET_DELIMITER primary)* RBRACE
+SetInitializer
+    : LBRACE SetMembers RBRACE
+    ;
+
+fragment
+SetMembers
+    : SetLiteral (SPACE* COMMA SPACE* SetLiteral)*
```

Comment:
```suggestion
    : literal (SPACE* COMMA SPACE* literal)*
```

Let's use the same literal. Also, can we use `primary` as it used to be used? Perhaps sets of sets would be appropriate.

Reply:
Maybe in future? I do not want to complicate this any further for now. We have documented this since the beginning without supporting it.